app/assets/javascripts/mapping_rules.js

```diff
@@ -1,34 +0,0 @@
-$(document).ready(function () {
```

Comment:
ok, removing this or not depends on https://github.com/3scale/porta/pull/2274/files#r537618015